nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/serialization/FlowFromDOMFactory.java

```diff
@@ -431,6 +431,7 @@ public static ProcessorDTO getProcessor(final Element element, final StringEncry
         configDto.setYieldDuration(getString(element, "yieldPeriod"));
         configDto.setBulletinLevel(getString(element, "bulletinLevel"));
         configDto.setLossTolerant(getBoolean(element, "lossTolerant"));
+        configDto.setExecutionNodeRestricted(getBoolean(element, "executionNodeRestricted"));
```

Comment:
We shouldn't be looking for an "executionNodeRestricted" element in the flow here - it won't be there because this is not a configuration element.